### PR TITLE
Setup nightly evals runs

### PR DESCRIPTION
**Pull request by stiyyagura0901**
Setup nightly evals runs

### CODE REVIEW

File: braintrust/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 This is a Braintrust eval example that demonstrates how to run evals with the [`circleci/evals`](https://circleci.com/developer/orbs/orb/circleci/evals) orb.
 
+
```
Comment (stiyyagura0901) [inline braintrust/README.md:7]:
```suggestion
```

Comment (stiyyagura0901) [inline braintrust/README.md:7]:
```suggestion
```

File: .circleci/run_evals_config.yml [hunk 2]
```diff
@@ -26,12 +37,14 @@ workflows:
       - run-braintrust-evals:
           context:
             - ai-llm-eval-examples # Replace this with your context name
+            - slack-notification-access-token # Replace this with your context name where SLACK_ACCESS_TOKEN is stored
```
Comment (ryan-circleci) [inline .circleci/run_evals_config.yml:40]: Good idea...We'll update the docs accordingly for this option. I think it'll help the user.

**Review by ryan-circleci — APPROVED** [review]
🔥

**Pull request merged**